client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -134,12 +134,14 @@ export const yearCallback = (value: string, { section }: CaseCallbackParams) =>
   section.year = yearCase(value);
 };
 
-export const studentHoursCallback = (value: string, { course }: CaseCallbackParams) => {
-  course.studentHours = numberDefaultZeroCase(value);
+export const studentHoursCallback = (value: string, { section }: CaseCallbackParams) => {
+  section.studentHours = value.startsWith("$")
```

Comment:
This was a little bug with excel I was getting that was prepending $ to the student and faculty hours. I figured it'd be good just to take it out since professors may have a similar issue